pkg/cmd/server/server.go

```diff
@@ -57,15 +57,15 @@ import (
 	"github.com/heptio/ark/pkg/util/kube"
 )
 
-func NewCommand() *cobra.Command {
+func NewCommand(baseName string) *cobra.Command {
 	var kubeconfig string
 
 	var command = &cobra.Command{
 		Use:   "server",
 		Short: "Run the ark server",
 		Long:  "Run the ark server",
 		Run: func(c *cobra.Command, args []string) {
-			s, err := newServer(kubeconfig)
+			s, err := newServer(kubeconfig, baseName+"-server")
```

Comment:
nit: you could just use c.Parent().Name() and c.Name() to construct this rather than passing in the baseName and hardcoding "server"